src/shims/ApiCompatBaseline.uap.netstandardOnly.txt

```diff
@@ -120,4 +120,23 @@ CannotSealType : Type 'System.Linq.EnumerableExecutor' is effectively (has a pri
 MembersMustExist : Member 'System.Linq.EnumerableExecutor..ctor()' does not exist in the implementation but it does exist in the contract.
 CannotSealType : Type 'System.Linq.EnumerableQuery' is effectively (has a private constructor) sealed in the implementation but not sealed in the contract.
 MembersMustExist : Member 'System.Linq.EnumerableQuery..ctor()' does not exist in the implementation but it does exist in the contract.
-Total Issues: 121
+
```

Comment:
nit: Extra line (Not sure if its intentional)

Reply:
yeah, I was trying to separate the additions I made for the temporary workaround so that I'll notice the diff when we get the new API compat.